packages/babel-parser/test/fixtures/typescript/class/accessor/input.ts

```diff
@@ -0,0 +1,21 @@
+abstract class Foo {
+  accessor prop: number = 1;
+  static accessor prop2: number = 1;
+  accessor #prop3: number = 1;
+  accessor [prop4]: number = 1;
+  private accessor prop5: number = 1;
+  abstract accessor prop6: number;
+  declare accessor prop7: number;
+  private accessor #p: any;
+
+  accessor a!;
+  abstract accessor #s;
+  accessor #d?;
```

Comment:
This is not allowed in an abstract context. Can you update the test case?

Reply:
Are you saying `abstract accessor #s;`?

Reply:
Oh actually `accessor #d?` is not allowed in either abstract class or non-abstract class. It seems to me the test case `class/accessor` covers only valid cases, can we make it a separate invalid test case?